js/login.js

```diff
@@ -0,0 +1,97 @@
+const emailInput = document.getElementById('email')
+const passwordInput = document.getElementById('password')
+const loginButton = document.querySelector('.login-form__button')
+const eyeicon = document.querySelector('.login-form__input--eye-icon')
+
+//Email Validation
+function validateEmail(email) {
+  const regex = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+/;
+  return regex.test(email);
+}
+
+//Password Validation
+function validatePassword(password) {
+  return password.length >= 8;
```

Comment:
8 이라는 상수값도 따로 어떤 수인지를 알기 쉽게 상수로 정의해주시면 좋을 거 같아요!
```suggestion
  const MINIMUM_PW_LENGTH = 8
  
  ....
  
  
  return password.length >= MINIMUM_PW_LENGTH;
```